src/Package/Impl/Repl/VsRInteractiveWindowComponentContainerFactory.cs

```diff
@@ -14,39 +15,67 @@
 using Microsoft.VisualStudio.Shell;
 using Microsoft.VisualStudio.Shell.Interop;
 using Microsoft.VisualStudio.Utilities;
+using static System.FormattableString;
 
 namespace Microsoft.VisualStudio.R.Package.Repl {
     [Export(typeof(IInteractiveWindowComponentContainerFactory))]
     internal class VsRInteractiveWindowComponentContainerFactory : IInteractiveWindowComponentContainerFactory {
         private readonly Lazy<IVsInteractiveWindowFactory> _vsInteractiveWindowFactoryLazy;
         private readonly IContentTypeRegistryService _contentTypeRegistryService;
+        private IRInteractiveWorkflow _interactiveWorkflow;
+        private IVsWindowFrame _frame;
```

Comment:
Probably not (at least not eventually with multiple REPLs). Let me see what I can do.